FPSCamera/InstanceCameras/VehicleCamera.cs

```diff
@@ -140,7 +140,41 @@ public void GetInstanceSpeed(Vector3 position)
             Vector3 velocity = v.GetSmoothVelocity(followInstance);
             FPSCameraSpeedUI.Instance.speed = velocity.magnitude;
             FPSCameraSpeedUI.Instance.destinationName = GetDestination();
-            FPSCameraSpeedUI.Instance.streetName = RaycastRoad(position);
+
+            ushort firstVehicle = vManager.m_vehicles.m_buffer[(int)followInstance].GetFirstVehicle(followInstance);
+            VehicleInfo info = vManager.m_vehicles.m_buffer[firstVehicle].Info;
+
+            if (info.GetService() == ItemClass.Service.PublicTransport && FPSCamera.instance.config.showPassengerCount)
+            {
+                FPSCameraSpeedUI.Instance.passengersOrStreet = GetPassengerNumbers();
+                if (info.m_vehicleType != VehicleInfo.VehicleType.CableCar)
+                {
+                    try
+                    {
+                        GetLastStopExchange();
+                    }
+                    catch (System.IO.FileNotFoundException)
```

Comment:
Is the `FileNotFoundException` from calling IPT2? If so, I wonder if there's any benefit from doing the mod check on load instead, since [there may be performance implications with exceptions everywhere](https://stackoverflow.com/questions/891217/how-expensive-are-exceptions-in-c), and this is a "on every frame" operation IIRC

Reply:
Yes it is. Doing the mod check on load is most likely better, this was the only way I got it working. I have no idea how to do it another way, cities modding is very new to me.

Reply:
I found a way! I added a check on startup.